engine/setup/src/c/setup.cpp

```diff
@@ -160,11 +161,20 @@ int main( int argc, char *argv[] )
             //Test if the dir exist and contains config_file
             bogus_int = chdir( origpath );
             bogus_int = chdir( (*vsit).c_str() );
-            FILE *setupcfg = fopen( "setup.config", "r" );
+            FILE* setupcfg;
+#ifdef WIN32
+            fopen_s(&setupcfg, "setup.config", "r");
+#else
+            setupcfg = fopen("setup.config", "r");
+#endif
```

Comment:
`fopen` should work on Windows too just fine. I know, MSVC complains about security when not using their special `_s` functions, but let's keep to cross-platform standards as much as we can. (The `_s` junk doesn't really buy you anything; just ties stuff to MSVC.)